### DISCUSSION TITLE
Blob, Its issues and some ideas on how to fix them.

### DISCUSSION THREAD

**Discussion by hal9000PR** [ParadiseSS13/Paradise#19640] [Documents]
## Issues
The numbers next to the issue relates to the potential fixes.

- Blob is unfun to fight. (1)
You stand diagonal from the blob tiles and weld them. This is what the majority of the crew does. it feels slow, and boring, and alone it feels like you make little to no progress.

- Blob success is decided very quickly. (2)
Either the blob is discovered before it can split and it loses, or it gets a good economy started and can fend for itself.

- Splitting is almost mandatory to win. (3)
Its never a good idea to delay splitting ever, splitting doubles your economy and your eyes on flanks.

- Spore spam is unavoidable chip damage. (4)
Its annoying and unfun to fight against.

- People can (and do) opt not to fight it. (5)
Due to blob having zero immediate effect on anyone outside of its boarders + 7 tiles, people can just avoid it.

- Certain weapons are too effective against it. (6)
Flashbangs, accelerator lasers, (xrays?), KAs, and KCs (against blobbers). As a result blob can be "cheesed" 

- Blob spawning. (7)
It can happen far too late into the round, and feels slightly too common still. contributing to the unfun to fight point.

## Potential Solutions.
(2) Give blobs periodic resources.
Make resource generation based on a cos function, so it starts high, then drops then increases, etc. and then buff the base generation rate. 
why? This means blob gets set up faster, meaning its less dependant on not being spotted. It also gives the blob times of "weakness" that the crew can exploit, even if they are unknowing of when the blob is "weak" 

(1) Reduce the health of blob tiles, Increase the rate of passive expansion. 
Hopefully, this will make fighting a blob feel like you are getting further even if you arent.

(1)(4) Reduce the rate of blob spores spawning, buff them. Add a minispore turret
Increase their HP, damage, etc. including when a zombie. when they die, make it explode after a delay, giving a tell beforehand. make the explosion MUCH more deadly. injecting more chems and more damage upfront. 

The spore turret would periodically fire out spores that would inject chems over a small area when the projectile hits. 

(5) Spore artillery
Giving blob a high cooldown, long range attack could do wonders. The effect doesnt have to be huge, it could be built onto the minispore turret. but it has to have some effect. 

(6) Weapon rebalances.
Nerf explosive weapons vs blob (KA), buff reflectives to 100% against energy weapons. whatever else people feel is necessay.

(3) Buff blob without splitting, make splitting have some downside to the blob.
really unsure how to go about this right now. but this is why its a discussion.

(7) Prevent spawning too late into the round.
I think Biohazard rolls should re-roll after 1:30 into the round.

(7)(1?) Reduce blob roll weight.
Maybe only temporarily. Could even be disabled temporarily, give the community a break from it, might make it more fun for people.

**Comment by FloFluoro** [comment]
I really like the spore artillery idea. Giving the blob a way to influence areas outside of its immediate range that isn't just "M3 blob spores offscreen and say a hail mary" would do wonders for encouraging crew to actually confront it.

**Comment by Sirryan2002** [comment]
I like the idea of making biohazards much less likely to spawn or not spawn at all when the round is nearing the end. Any biohazards that spawn around the 1:45 mark generally just get circumvented by the shuttle call or feel like they're butting in on existing round events.

A lot of issues with the blob I feel stem from problems (2) and (1) in tandem because, at a point, the crew knows for sure it is game over because it's actually completely impossible to fight back against the blob just in terms of DPS being completely outweighed by blob expansion unless you get lucky with a core snipe. 

I feel like a change in the goals for the blob is also necessary, "expand" is the name of the game for the blob, but endless expansion will become too big for players to feel like they can fight. So I think the blob needs some form of progression like having to "evolve" into different tiers where it loses and gains abilities, but when transitioning tiers, the blob becomes much weaker. I think this would contribute to more back-and-forth fights and a way to punish blobs that "get too big." This may have the secondary benefit of reducing CPU load that 600+ tile blobs create.

**Reply by hal9000PR** [reply]
I think blob hitting a "critical mass point" at which point they start to grow massively and basically win would be a good idea.

**Comment by warriorstar-orion** [comment]
You are spot on about the problems here as someone who has played against blob but never played it. I can't comment much as to balance but I wanted to ask if the turrets are AI-controlled. I'm positive we've discussed this before but AI turrets are expensive computationally because they have large lines of sight and scan all living things to determine threat priority. It's probably possible to short-circuit some of the existing logic for a blob-specific turret but yeah. Add a dozen crew, half dozen spores, a few nauts, and things pile up. 

EDIT: also spore buffs make me happy

**Comment by Coolrune206** [comment]
It would be nice if Blobbernauts were given a little more identity too than just high-HP, low damage walls. Maybe giving them forms they can shift between for HP, speed, and damage (with some delay to transforming, or needing to be on a blob tile) so they can be a little bit more than just walk forward slowly, click melee attackers, get two-shot by Miner with infinity armor + Kinetic Crusher.
Speaking of, I definitely think they could use some armor piercing to handle just how effective Riot/Mining armor is against them.

**Comment by Eric6426-Blossom** [comment]
Personally there's a fair few things I dislike, and have heard others dislike from the blob roundtype, I'll go on a few of those first;

The ending, its anti-climactic, when crew win it makes sense (Cores killed, roundend, lovely) but the Blob win is utterly anti-climactic. It simply reaches a number of tiles upon which the server ends the round - I feel that the idea that hitting crit-mass would simply make Blob expand too fast and have too many resources to kill would be better, while simply giving crew nuke codes or DS at worst - I don't see a reason why the roundtype needs a hard end on both teams, there's often always an admin to at-least force a DS if need be, and it makes for much better stories than "The blob was won! - The round is over!"

The start - Faerie already took care of my biggest issue with this, which is the fact that you don't get a timer just text prompts about it - However I do feel like blob should, maybe, after a bit (say like 5 minutes) be allowed to press a button to pop early, or let it last for a while so they can do some planning/blocking, and maybe forced to pop at around 8-15 minutes in? Gives them time to rush some quick sabotage whilst not enough to do anything too major

I'm honestly not sure if its possible to make blob also expand or attack sideways, but being able to simply sit there and click it has always been a major reason why Blob can just be overwhelmed - Lower health tiles might aggravate that problem, so I'd hope they could attack sideways

On the points posted as potential solutions, I believe they'd all be helpful in their own ways, more uniqueness to blob would be something interesting rather than it being how it is as of now.

On the note of blob starting, it could be given more 'power' than just being a carrier - Seeing how smart players and mice play as infected, they could maybe be given the ability to put 'blob' barricades or such before popping, for roundstart/midround maybe (even if the mice is kind of odd, maybe spore-traps?) so they can also prevent the very comical "Assistant found the core early and weldered it to death" moments alongside the buff to starting resources.

That's all I can think of right now, take my ideas with a grain of salt, but thanks for reading :p
